demo/config.yaml

```diff
@@ -31,8 +31,35 @@ lcr-modules:
             sample_fastq_2: "data/{sample_id}.read2.fastq.gz"
 
     vcf2maf:
+        dirs:
+            _parent: "__UPDATE__"
         inputs:
             vep_cache: "reference/vep_caches/"
+            sample_vcf_gz: "__UPDATE__"
+            convert_coord: "{SCRIPTSDIR}/crossmap/1.0/convert_maf_coords.sh"
+        vcf_base_name: "__UPDATE__"
+        options:
+            vcf2maf: "--filter-vcf 0 --vcf-tumor-id TUMOR --vcf-normal-id NORMAL --cache-version 86"
+            species: "homo_sapiens"
+        conda_envs:
+            vcf2maf: "{MODSDIR}/envs/vcf2maf-1.6.18.yaml"
+            crossmap: "{SCRIPTSDIR}/crossmap/1.0/convert_maf_coords.yaml"
+        # here you can specify path to txt file with a list of custom ENST IDs that override canonical selection
+        # it will be parsed to --custom-enst flag of vcf2maf
```

Comment:
I will make sure the demo configuration is maintained. I will leave that file empty, so nothing will be substituted. But will keep the file in the repo, just to show how it can be used, what do you think? 
I thought too that the list just goes to the config. But vcf2maf complains with an error that the file does not exist. So if I just enter ENST IDs, it is looking for that file. That is how I figured it should be a separate file and used it in the config.

Reply:
Sounds good